### ISSUE TITLE
rsbuild icon

### ISSUE THREAD

**Issue by StopNGo** [simple-icons/simple-icons#12007]
### Brand Name

rsbuild

### Website

https://rsbuild.dev/

### Popularity Metric

rsbuild rank is 391,687 https://similarweb.com/website/rsbuild.dev

### Official Resources for Icon and Color

SVG - https://assets.rspack.dev/rspack/rspack-logo.svg
Color - 240 150 91

**Comment by pvcresin** [comment]
Maybe SVG: https://github.com/rspack-contrib/rstack-design-resources/blob/main/rsbuild/rsbuild-logo.svg

**Comment by StopNGo** [comment]
> Maybe SVG: https://github.com/rspack-contrib/rstack-design-resources/blob/main/rsbuild/rsbuild-logo.svg

Yes, you are right. They have bunch of logos for their products. Would be greate to add all of tthem.

**Comment by PeterShaggyNoble** [comment]
No reason we can't add them all, @StopNGo, provided each one qualifies under our metrics of popularity. Please open a separate issue for each, though.

**Comment by StopNGo** [comment]
I understand. Thanks!
